index.html

```diff
@@ -1,49 +1,93 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="pages/index.css">
-    <title>Pink App</title>
-  </head>
-  <body class="page">
-    <div class="page__container">
-      <section class="research">
-        <h2 class="visually-hidden">Исcледование приложения</h2>
-        <picture>
-          <source srcset="images/research-timer-tablet.png" media="(max-width: 660px)">
-          <img src="images/research-timer-desktop.png" alt="Картинка таймера" class="research__image">
-        </picture>
-        <div class="research__inner-wrapper">
-          <p class="research__title">Эффект на&nbsp;24&nbsp;часа!</p>
-          <p class="research__paragraph">
-            Британские ученые провели исследования и доказали, что положительный
-            эффект от использования приложения длится целые сутки!
-          </p>
-          <a href="#" target="_blank" class="research__link">Узнать больше об исследовании</a>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="pages/index.css">
+  <title>Pink App</title>
+</head>
+
+<body class="page">
+  <div class="page__container">
+
+    <section class="research">
+      <h2 class="visually-hidden">Исcледование приложения</h2>
+      <picture>
+        <source srcset="images/research-timer-tablet.png" media="(max-width: 660px)">
+        <img src="images/research-timer-desktop.png" alt="Картинка таймера" class="research__image">
+      </picture>
+      <div class="research__inner-wrapper">
+        <p class="research__title">Эффект на&nbsp;24&nbsp;часа!</p>
+        <p class="research__paragraph">
+          Британские ученые провели исследования и доказали, что положительный
+          эффект от использования приложения длится целые сутки!
+        </p>
+        <a href="#" target="_blank" class="research__link">Узнать больше об исследовании</a>
+      </div>
+    </section>
+    <section class="reviews">
+      <h2 class="visually-hidden">Комментарии</h2>
+      <div class="slider">
+        <button class="slider__arrow-button slider__arrow-button_direction_left" aria-label="Previous Review">
```

Comment:
@MarPostovik Next up, let take a look at each individual arrow button.

When considering this element, it's important to note that our button serves as the interactive component, while the SVG image inside it is purely _decorative_. That's why we set the SVG attributes `aria-hidden="true"` and `focusable="false"`, ensuring that screen reader users will skip navigating into the SVG. However, we also need to ensure that keyboard users can focus on the button and perceive changes in styles accordingly.

Please follow these steps to enhance the functionality of our arrow buttons:
1. Remove the `opacity="0.3"` attribute from the `<path>` tag for both arrow buttons. SVG's path element is used to draw shapes, lines, and curves. No worries about how it end up looking, when we are done it will inherit color/opacity from SVG and look exactly how we need it to be.

2. Remove the `fill` attributes from the `<svg>` and `<path>` tags and instead assign an element with a class name `slider__arrow-icon` with `fill: currentColor;`, that will make SVG to take on the color from its parent element.

3. Transfer the `color`, `transition`, and `cursor` properties, along with the `:hover` and `:active` states, from the `slider__arrow-icon` to the `slider__arrow-button` element. Since the button itself is the interactive element, these styles should be applied directly to it.

Here's how it works now: When you hover over or click on a button, it applies corresponding styles. The icons' `fill` property, set to `currentColor`, allows the SVG to inherit its color from the parent element. Additionally, the `<path>` retains its color from the SVG, ensuring that the shape looks exactly as intended.

Here's how the updated code should look:
```html
<!-- index.html -->
<button class="slider__arrow-button slider__arrow-button_direction_left" aria-label="Previous Review">
  <svg class="slider__arrow-icon" aria-hidden="true" focusable="false" viewBox="0 0 22 41" xmlns="http://www.w3.org/2000/svg">
    <path d="M2.0413 20.5035L21.7483 1.4144C21.8281 1.3379 21.8914 1.24683 21.9346 1.14646C21.9778 1.04608 22 0.938398 22 0.829635C22 0.720871 21.9778 0.613187 21.9346 0.512814C21.8914 0.41244 21.8281 0.321369 21.7483 0.24487C21.6686 0.167255 21.5738 0.105671 21.4694 0.0636507C21.365 0.0216304 21.2532 0 21.1401 0C21.0271 0 20.9153 0.0216304 20.8109 0.0636507C20.7065 0.105671 20.6117 0.167255 20.532 0.24487L0.252535 19.9088C0.17249 19.9862 0.108978 20.0781 0.0656429 20.1793C0.0223074 20.2805 0 20.389 0 20.4985C0 20.6081 0.0223074 20.7166 0.0656429 20.8178C0.108978 20.919 0.17249 21.0109 0.252535 21.0883L20.532 40.7621C20.6967 40.9157 20.9167 41.001 21.1453 41C21.3139 40.9985 21.4783 40.9488 21.6181 40.8572C21.7578 40.7656 21.8666 40.6362 21.9309 40.485C21.9952 40.3339 22.0122 40.1677 21.9797 40.0072C21.9473 39.8467 21.8668 39.6991 21.7483 39.5827L2.0413 20.5035Z"/>
  </svg>
</button>
```

```css
/* slider__arrow-button.css */
.slider__arrow-button{
    position: absolute;
    top: 29px;
    padding: 0;
    color: rgb(40, 54, 69);
    border: none;
    background: none;
    cursor: pointer;
    transition: all .7s ease;
}

.slider__arrow-button:hover{
  color: rgb(126 134 143);
}

.slider__arrow-button:active{
  color: rgb(233 235 236);
}

/* media rules */
```

```css
/* slider__arrow-icon.css */
.slider__arrow-icon{
    width: 22px;
    height: 41px;
    fill: currentColor;
}
```

Reply:
Done ✅